checkov/terraform/checks_infra/solvers/attribute_solvers/base_attribute_solver.py

```diff
@@ -7,50 +10,59 @@
 
 
 class BaseAttributeSolver(BaseSolver):
-    operator = ''
+    operator = ""
 
-    def __init__(self, resource_types, attribute, value):
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
```

Comment:
@gruebel Why is `attribute` optional? I believe it should be required here

Reply:
as I mentioned in the call, this is related to connection solver, where `attribute` is `None`.